app/main/views/audits.py

```diff
@@ -115,6 +115,27 @@ def list_audits():
     elif object_id:
         abort(400, 'object-id cannot be provided without object-type')
 
+    after_event_id = request.args.get('after-event-id')
```

Comment:
Ha! Fine, I can kind of see that, and as we have an assortment here anyway it probably doesn't matter.